sql/pr/time-to-merge-per-size.sql

```diff
@@ -1,44 +1,38 @@
 -- Time to merge per PR size
 WITH
-stats AS (
+ranked_stats AS (
     SELECT
-        commit_id
-      , sum(added_lines) + sum(deleted_lines) AS size
-    FROM git.default.diff_stats
-    GROUP BY 1
+        pull_number
+      , additions + deletions AS size
+      , row_number() OVER (PARTITION BY pull_number ORDER BY updated_at DESC) AS freshness_rank
+    FROM pull_stats
+)
+, stats AS(
+    SELECT
+      pull_number,
+      size
+    FROM ranked_stats
+    WHERE freshness_rank = 1
```

Comment:
We should use the unique_pull_stats view here

Reply:
Our unique views didn't handle rows with the same `updated_at` correctly, so I replicated this approach there and now it's fixed. What I'm asking here is to use these views for consistency, since we do in other queries.